src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -118,6 +118,36 @@
 
         $(SELECTOR_MODAL_BULK_ACTION_FAIL).modal('show');
     };
+    const refreshTrashModal = (event) => {
```

Comment:
I wonder, maybe we should put these changes here:
https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/src/bundle/Resources/public/js/scripts/admin.trash.js